### PR TITLE
Fixes adminwarn spam after rigged cell explosions

### LINKED ISSUE #13242: Plasma cell explosions continuously spam admin logs after the explosion passes
ADMIN: LOG: Rigged power cell explosion, last touched by Harse Theef
ADMIN: LOG: Rigged power cell explosion, last touched by Harse Theef
ADMIN: LOG: Rigged power cell explosion, last touched by Harse Theef
ADMIN: LOG: Rigged power cell explosion, last touched by Harse Theef

Doesn't seem to happen with every power cell explosion

### PR DESCRIPTION
**Pull request by Sprok0**
Fixes #13242

### CODE REVIEW

**Comment by gbasood** [comment]
Uh
This seems like a workaround fix
Why is it warning them repeatedly in the first place?

**Comment by Sprok0** [comment]
I've got no idea, I assume because it caused too small of an explosion to destroy itself, and didn't delete properly.

File: code/modules/power/cell.dm [hunk 1]
```diff
@@ -107,8 +107,8 @@
 	message_admins("LOG: Rigged power cell explosion, last touched by [fingerprintslast]")
 
 	explosion(T, devastation_range, heavy_impact_range, light_impact_range, flash_range)
-
-	spawn(1)
+	if(!gcDestroyed)
```
Comment (gbasood) [inline code/modules/power/cell.dm]: I'm not sure this check needs to be here.
If we get rid of it, charge is set to 0, and the `if (charge==0)` check will cause it to return instantly anyway, so it won't qdel itself again

**Comment by Sprok0** [comment]
I tested a bit more, and what's happening is that a low-power cell makes like an 0,0,1 explosion, deletes itself, but whatever was using it still has a reference to it and keeps trying to use it. This doesn't fix that, but at least kills the spam.

File: code/modules/power/cell.dm [hunk 2]
```diff
@@ -109,6 +109,7 @@
 	explosion(T, devastation_range, heavy_impact_range, light_impact_range, flash_range)
 
 	spawn(1)
```
Comment (gbasood) [inline code/modules/power/cell.dm]: Nah, remove the spawn
I don't know why it's here in the first place
And actually set charge to 0 before calling explosion()

**Pull request merged**